todo_app/templates/index.html

```diff
@@ -2,17 +2,29 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
+<form action="/item" method="post" class="todo-form" id="todoForm">
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="todo-header" id="todoHeader">To-Do App</h1>
+    <p class="todo-header-para" id="todoHeaderPara">Things to-do app.</p>
   </div>
-
+  <!-- <div class="jumbotron"> -->
+    <h2 class="add-item-input add-item-submit" id="addItemInputSubmit">Add Item</h1>
```

Comment:
this really is very minor but this line has an `<h2>` and a `</h1>` which doesn't match up.

It looks like flask actually corrects this while rendering it, but I'd still recommend fixing it in case this causes any weird bugs in the future!

Reply:
Thanks very much - have fixed in commit a943886